simtools/model/telescope_model.py

```diff
@@ -571,125 +571,129 @@ def getTelescopeTransmissionParameters(self):
+    def _load_derived_values(self):
         """Load the derived values for this telescope from the DB."""
         self._logger.debug("Reading derived data from DB")
-        db = db_handler.DatabaseHandler(mongoDBConfig=self.mongoDBConfig)
-        self._derived = db.getDerivedValues(
+        db = db_handler.DatabaseHandler(mongo_db_config=self.mongo_db_config)
+        self._derived = db.get_derived_values(
             self.site,
             self.name,
-            self.modelVersion,
+            self.model_version,
         )
 
-    def _loadCamera(self):
+    def _load_camera(self):
         """Loading camera attribute by creating a Camera object with the camera config file."""
-        cameraConfigFile = self.getParameterValue("camera_config_file")
-        focalLength = self.getParameterValue("effective_focal_length")
-        if focalLength == 0.0:
+        camera_config_file = self.get_parameter_value("camera_config_file")
+        focal_length = self.get_parameter_value("effective_focal_length")
+        if focal_length == 0.0:
             self._logger.warning("Using focal_length because effective_focal_length is 0.")
-            focalLength = self.getParameterValue("focal_length")
+            focal_length = self.get_parameter_value("focal_length")
         try:
-            cameraConfigFilePath = gen.findFile(cameraConfigFile, self._configFileDirectory)
+            camera_config_file_path = gen.find_file(camera_config_file, self._config_file_directory)
         except FileNotFoundError:
             self._logger.warning(
-                "CameraConfigFile was not found in the config directory - "
+                "The camera_config_file was not found in the config directory - "
                 "Using the one found in the model_path"
             )
-            cameraConfigFilePath = gen.findFile(cameraConfigFile, self.io_handler.model_path)
+            camera_config_file_path = gen.find_file(camera_config_file, self.io_handler.model_path)
 
         self._camera = Camera(
-            telescopeModelName=self.name,
-            cameraConfigFile=cameraConfigFilePath,
-            focalLength=focalLength,
+            telescope_model_name=self.name,
+            camera_config_file=camera_config_file_path,
+            focal_length=focal_length,
         )
 
-    def _loadSimtelConfigWriter(self):
-        if not hasattr(self, "simtelConfigWriter"):
-            self.simtelConfigWriter = SimtelConfigWriter(
+    def _load_simtel_config_writer(self):
+        if not hasattr(self, "simtel_config_writer"):
+            self.simtel_config_writer = SimtelConfigWriter(
                 site=self.site,
-                telescopeModelName=self.name,
-                modelVersion=self.modelVersion,
+                telescope_model_name=self.name,
+                model_version=self.model_version,
                 label=self.label,
             )
 
-    def isFile2D(self, par):
+    def is_file_2D(self, par):
```

Comment:
I would put 2D also as 2d for the sake of consistency and keep as it is in the text below

Reply:
I would like to keep it as 1D and 2D. I know it's not consistent, but since it's just one letter and the convention of 1D and 2D is so prevalent, I think it makes sense.